modules/reference/pages/feature/socialLogin/examples.adoc

```diff
@@ -126,13 +127,13 @@ The following example configures Instagram as the social media provider::
 </oauth2Login>
 ----
 
-// LC: Leaving the following links in the source for now to show where this topic should link to when the relevant equivalent topics are published in the Open Liberty docs (do not link to the KC from Open Liberty docs). Remove this commented section when the relevant links are added in future.
+// LC: Leaving the following links in the source, for now, to show where this topic should link to when the relevant equivalent topics are published in the Open Liberty docs (do not link to the KC from Open Liberty docs). Remove this commented section when the relevant links are added in the future.
```

Comment:
these comments on lines 130-132 are out of date and should be deleted

Reply:
Deleted 130-132